app/views/index.html

```diff
@@ -1,9 +1,5 @@
-{% set title = 'Home' %}
-{% set navigation = 'transactional' %}
-{% from 'components/callout/macro.njk' import callout %}
-{% extends 'includes/site-layout.html' %}
-
-{% block body %}
+{% set title = 'Home' %} {% set navigation = 'transactional' %} {% from 'components/callout/macro.njk' import callout %}
+{% extends 'includes/site-layout.html' %} {% block body %}
```

Comment:
Just wondering if there's a particular reason for having these in-line?

Reply:
It's because I did a format document in vs code .